Resources/Prototypes/Entities/Objects/Specific/Hydroponics/seeds.yml

```
@@ -495,6 +495,17 @@
     - type: Sprite
       sprite: Objects/Specific/Hydroponics/ambrosia_deus.rsi
 
+- type: entity
+  parent: SeedBase
+  name: packet of ambrosia horribilis seeds
+  description: "A plant spawned from the devil himself, nothing good can come of it."
```

```suggestion
  description: A plant spawned from the devil himself, nothing good can come of it.
```
no need for quotation marks